### PR TITLE
update libbeat fields.ecs.yml file and ecsVersion to 1.10.0

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to update the `fields.ecs.yml` file in libbeat, which comes from the ECS repo: https://github.com/elastic/ecs/blob/1.10/generated/beats/fields.ecs.yml
Also updated `ecsVersion` variable from 1.9.0 to 1.10.0.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

**Comment by kaiyan-sheng** [comment]
flaky test: https://github.com/elastic/beats/issues/26181

**Comment by kaiyan-sheng** [comment]
CI failed because of https://github.com/elastic/beats/issues/26239.

**Comment by kaiyan-sheng** [comment]
heartbeat CI failure: https://github.com/elastic/beats/issues/26205

File: packetbeat/protos/http/event.go [hunk 1]
```diff
@@ -36,6 +36,9 @@ type ProtocolFields struct {
 	// "Lowercase Capitalization" in the "Implementing ECS"  section.
 	RequestMethod common.NetString `ecs:"request.method"`
 
+	// HTTP request ID.
+	RequestID common.NetString `ecs:"request.id"`
```
Comment (P1llus) [inline packetbeat/protos/http/event.go:40]: This one was intended right?

**Review by P1llus — APPROVED** [review]
LGTM, one small comment, I remember you told me it was needed to get the PR to pass, but was unsure.

File: packetbeat/protos/http/event.go (hunk 1 above)
Comment (kaiyan-sheng) [reply packetbeat/protos/http/event.go:40]: Yes, it's required for unit test to pass :)

**Pull request merged**